pyingest/parsers/affils.py

```diff
@@ -83,6 +83,20 @@ def parse(self):
             new_string = new_string.strip()
             new_string = new_string.strip(';')
             new_string = new_string.strip()
+            # look for duplicate emails here
+            new_aff_array = list()
+            aff_array = new_string.split(';')
```

Comment:
I think this should be more efficiently done as such:
   aff_array = [x.strip() for x in new_string.split(';')]
   new_aff_array = list(dict.fromkeys(aff_array)) # python3 guarantees order preserved
   new_string = '; '.join(new_aff_array)